Gemfile

```diff
@@ -14,5 +14,6 @@ gem 'chef', chef_version
 gem 'chef-sugar'
 gem 'chef-vault', chef_vault_version
 gem 'chefspec', chefspec_version
+gem 'cucumber-core', '~> 3.2'
```

Comment:
Add https://github.com/cucumber/cucumber-ruby-core/issues/160 as a comment so that we know why we are locking this.